recipes/boost/all/conanfile.py

```diff
@@ -137,7 +137,6 @@ def _min_compiler_version_default_cxx11(self):
         return {
             "gcc": 6,
```

Comment:
The default mode is C++98 for GCC versions prior to 6.1, and C++14 for GCC 6.1 and above.

Reply:
#Minimum compiler version having c++ standard >= 11 at 135

 # Minimum compiler version uses c++ standard >= 11 by default